Feature/ehr 150 ehr status frontend

jakesmolka opened the pull request:
**WIP**: swagger UI problem needs to be discussed

Collective PR for
- EHR API - EHR_STATUS endpoints "frontend"
- Implement Get EHR_STATUS by version id
- Implement Update EHR_STATUS
- Implement Get EHR_STATUS version by time


closes ehrbase/project_management#11 ehrbase/project_management#12 ehrbase/project_management#13 ehrbase/project_management#14

Geisterfalle commented:
Some remarks while testing:

An URI like `localhost:8080/ehrbase/rest/openehr/v1/ehr/784912f3-def7-4f87-bc69-840c27594c17/ehr_status` should provide the latest version of the EHR_STATUS. Instead, the following message is shown:

`{
  "timestamp": 1571949951287,
  "status": 400,
  "error": "Bad Request",
  "exception": "org.springframework.web.bind.UnsatisfiedServletRequestParameterException",
  "message": "Parameter conditions \"version_at_time\" not met for actual request parameters: ",
  "path": "/ehrbase/rest/openehr/v1/ehr/784912f3-def7-4f87-bc69-840c27594c17/ehr_status"
}`

jakesmolka commented:
@Geisterfalle 
1. Fixed in upcoming commit.
2. Wasn't in the scope of these issues. Wouldn't touch it now, because of the little time left on this sprint. We should create a separate issue for that.
3. Fixed in upcoming commit.
